### ISSUE TITLE
Very low accuracy for small interval

### ISSUE THREAD

**Issue by pagenum** [tna0y/Python-random-module-cracker#7]
```
import random, time
from randcrack import RandCrack

random.seed(10)

rc = RandCrack()

for i in range(624):
	rc.submit(random.randrange(0,5))
	# Could be filled with random.randint(0,4294967294) or random.randrange(0,4294967294)

for i in range(20):
	print("Random result: {}\nCracker result: {}".format(random.randrange(0,5), rc.predict_randrange(0,5)))


```

```
Random result: 4
Cracker result: 2
Random result: 4
Cracker result: 4
Random result: 0
Cracker result: 2
Random result: 3
Cracker result: 0
Random result: 2
Cracker result: 0
Random result: 3
Cracker result: 4
Random result: 1
Cracker result: 0
Random result: 0
Cracker result: 0
Random result: 3
Cracker result: 0
Random result: 3
Cracker result: 0
Random result: 0
Cracker result: 4
Random result: 0
Cracker result: 2
Random result: 3
Cracker result: 0
Random result: 1
Cracker result: 0
Random result: 0
Cracker result: 0
Random result: 3
Cracker result: 0
Random result: 4
Cracker result: 4
Random result: 0
Cracker result: 0
Random result: 0
Cracker result: 4
Random result: 4
Cracker result: 2
Press any key to continue . . .
```

Only 25% accuracy.

**Comment by tna0y** [comment]
Please try randcrack from the master branch of thip repository. The fix has not been uploaded to the PyPI yet.

**Comment by tna0y** [comment]
The fix has been uploaded to PyPI in version 0.2.0.
